internal/manifests/config.go

```diff
@@ -71,6 +75,10 @@ func ConfigOptions(opt Options) config.Options {
 			QuerierCPULimits:      opt.ResourceRequirements.Querier.Requests.Cpu().Value(),
 			QueryFrontendReplicas: opt.Stack.Template.QueryFrontend.Replicas,
 		},
+		WriteAheadLog: config.WriteAheadLog{
+			Directory:             strings.TrimRight(walDirectory, "/"),
```

Comment:
What's the significance of using `strings.TrimRight()` here?
The constant `walDirectory` already equals `/tmp/wal`. There is no `/` at the end to trim.

Reply:
That was the result of copy & paste.